@types/state-machine.d.ts

```diff
@@ -0,0 +1,31 @@
+// TypeScript Version: 3.7.4
+
+export default StateMachine
+
+declare class StateMachine {
+    constructor(machine: MachineDefinition, initialState?: string);
+    currentState: string;
+    getDotFile(): string;
+    transition(transition: string, data?:any):Promise<any>
+}
+
+export type MachineDefinition = {
```

Comment:
I exported `MachineDefintion` and `StateDefinition` below in case we need them in the implementation. I don't think anything else would need to be exported. 🤔

Reply:
yeah I think that makes sense, we can always patch it if we find we need more